Userland/Libraries/LibJS/Runtime/Temporal/ZonedDateTime.cpp

```diff
@@ -451,28 +449,26 @@ ThrowCompletionOr<DurationRecord> difference_zoned_date_time(GlobalObject& globa
 }
 
 // 6.5.7 NanosecondsToDays ( nanoseconds, relativeTo ), https://tc39.es/proposal-temporal/#sec-temporal-nanosecondstodays
-ThrowCompletionOr<NanosecondsToDaysResult> nanoseconds_to_days(GlobalObject& global_object, BigInt const& nanoseconds_bigint, Value relative_to_value)
+ThrowCompletionOr<NanosecondsToDaysResult> nanoseconds_to_days(GlobalObject& global_object, Crypto::SignedBigInteger const& nanoseconds_, Value relative_to_value)
 {
     auto& vm = global_object.vm();
 
-    // 1. Assert: Type(nanoseconds) is BigInt.
-
-    // 2. Set nanoseconds to ℝ(nanoseconds).
-    auto nanoseconds = nanoseconds_bigint.big_integer();
+    // NOTE: We need to reassign this later, and therefore can't use the reference.
+    auto nanoseconds = nanoseconds_;
```

Comment:
Can we just make the parameter a value instead of a reference then? It would make this slightly nicer and give future users the opportunity to save a copy

Reply:
Ah, that's a bit silly, yeah. Focused a bit too much on those references :^) Fixed!